src/endpoint-microservice/restapi/__tests__/restapi.controller.spec.ts

```
@@ -469,6 +487,105 @@ describe('restapi controller', () => {
+    });
+
+    it('GET /openapi.json should return 403 when preflight fails', async () => {
+      mockProxyCoreApiService.api.revision.mockResolvedValueOnce({
+        data: null,
+        error: { message: 'Forbidden', statusCode: 403 },
+        status: 403,
+      });
+
+      await request(app.getHttpServer())
+        .get(getOpenApiUrl())
+        .expect(403);
```

ESLint

[error] Replace `⏎········.get(getOpenApiUrl())⏎········` with `.get(getOpenApiUrl())`

(prettier/prettier)